scripts/ci-conformance.sh

```diff
@@ -36,7 +36,12 @@ cleanup() {
 }
 trap cleanup EXIT
 
-#Install requests module explicitly for HTTP calls
+# Set correct pip3.7 location for python3.
+if ! test -f /usr/bin/pip3.7 && test -f /usr/local/bin/pip3.7; then
+  ln -s /usr/local/bin/pip3.7 /usr/bin/pip3.7
```

Comment:
Actually, one other question. Shouldn't this just be `pip3` like `python3`? I might be wrong, but isn't pip3 for all pip3+ things? So like, if we want a different version, we don't have to come and update that?

Reply:
Of course, this is rather a mute point if pip3.7 is specifically required. :D

Reply:
That's true, sure. But `python3 -m pip` is calling `/usr/bin/pip3.7`. Maybe we have to set the right version somewhere? I'm not that familiar with that Debian image

Reply:
Yeah, I can see that. Interesting. Who and why moved Python away. :D That would be something that we would need to find out.

Reply:
The last time it worked is `v20220624-1a63fdd9f2-master`.


2437cd785ea7 | v20220624-1a63fdd9f2-master | 1.2 GB | 24 Jun 2022 | 24 Jun 2022
-- | -- | -- | -- | --



So something happened between 24 Jun and 8th of July.

I'm not saying we shouldn't do this PR, but I would rather find out what happened then do this and then get bitten again once python3 refers to 3.8 if that makes sense?

Reply:
I definitely agree with you. This is just a quick-fix to at least fix our CAPO conformance tests -- and maybe CAPA conformance tests.

Reply:
Honestly, I think this is a bug in the image building process.

Reply:
https://github.com/kubernetes/test-infra/commit/fa9fc389d33f92e9d41932d0c16dbe13ff8e9252

Added no-cache. That's the only pip related change and it somehow caused all of this.

Reply:
Op! Found it! :D 

https://github.com/kubernetes/test-infra/pull/24967

Reply:
Actually this hasn't been working for the infra image since 7th of June. This is the last working image: `v20220607-4b3fb50c36-test-infra`.

Reply:
Thanks for finding out, really appreciate your help! :)